core/string/ustring.h

```diff
@@ -520,7 +518,17 @@ class String {
 	String path_join(const String &p_file) const;
 	char32_t unicode_at(int p_idx) const;
 
+	void parse_utf32(const StrRange<char32_t> &p_cstr);
```

Comment:
I think this would be better suited under the `utf16` section, as that's the ordering generally used elsewhere.